demos/python_demos/speech_recognition_demo/README.md

```diff
@@ -18,16 +18,20 @@ The demo depends on the `ctcdecode_numpy` Python extension module,
 which implements CTC decoding in C++ for faster decoding.
 Please refer to [Using Open Model Zoo demos](../../README.md) for instructions
 on how to build the extension module and prepare the environment for running the demo.
+Alternatively, you can [install ctcdecode-numpy from OpenVINO 2021.1 Open Model Zoo](https://github.com/openvinotoolkit/open_model_zoo/tree/2021.1/demos/python_demos/speech_recognition_demo/ctcdecode-numpy#installation).
 
 ## Model preparation
 
-You can download and convert a pre-trained Mozilla\* DeepSpeech 0.6.1 model with
-OpenVINO [Model Downloader](../../../tools/downloader/README.md).
+You can download and convert a pre-trained Mozilla\* DeepSpeech 0.8.2 model with
+OpenVINO [Model Downloader](../../../tools/downloader/README.md) and the provided conversion scripts.
 This essentially boils down to the following commands:
 ```shell
-source <openvino_path>/bin/setupvars.sh
-<openvino_path>/deployment_tools/open_model_zoo/tools/downloader/downloader.py --name mozilla-deepspeech-0.6.1
-<openvino_path>/deployment_tools/open_model_zoo/tools/downloader/converter.py --name mozilla-deepspeech-0.6.1
+source <openvino_dir>/bin/setupvars.sh
```

Comment:
requiring manual step before call to conevrt.py is not acceptable. Have you noticed, that convert.py allows call customer script with predefined name `pre-convert.py` where you can put required pre-conversion operations. See for example as it implemented in [yolo-v4-tf](https://github.com/openvinotoolkit/open_model_zoo/tree/develop/models/public/yolo-v4-tf).

Reply:
Done.